src/Uno.Foundation/Extensibility/ApiExtensibility.cs

```diff
@@ -61,7 +65,8 @@ public static void Register<TOwner>(Type type, Func<TOwner, object> builder)
 	/// <param name="owner">An optional owner to be passed to the extension constructor</param>
 	/// <param name="instance">The instance if the creation was successful</param>
 	/// <returns>True if the creation suceeded, otherwise False.</returns>
```

Comment:
```suggestion
	/// <param name="owner">An optional owner to be passed to the extension constructor</param>
	/// <param name="instance">The instance if the creation was successful</param>
	/// <returns>True if the creation succeeded, otherwise False.</returns>
```